src/smc_mallows_new_users_complete.cpp

```diff
@@ -85,6 +85,9 @@ Rcpp::List smc_mallows_new_users_complete(
   arma::mat alpha_samples(N, (n_users + Time + 1));
   const arma::vec alpha_samples_0 = Rcpp::rexp(N, 1);
   alpha_samples.col(0) = alpha_samples_0;
+  
+  /* generate vector to store ESS */
+  arma::vec ESS_vec = (Time, arma::fill::zeros);
```

Comment:
I don't understand this constructor. Is `ESS_vec` supposed to be initialized with the integer `Time` or a bunch (how many?) zeros?

Also, please note that `Time` is an integer and `vec` denotes a double, so C++ may complain.

Reply:
`ESS_vec` is a row vector of length `Time`. You can initialise with zeros, but it can remain empty to begin with. The values to be stored in `ESS_vec` will be double type.

Would `rowvec ESS_vec(Time, fill::zeros);` or `rowvec ESS_vec(Time);` work?